crates/bevy_ecs/src/storage/blob_vec.rs

```diff
@@ -388,4 +386,11 @@ mod tests {
 
         assert_eq!(*drop_counter.borrow(), 6);
     }
+
+    #[test]
+    fn blob_vec_drop_empty() {
```

Comment:
```suggestion
    fn blob_vec_drop_empty_capacity() {
```
Could we specify the the `empty` part is in relation to the `capacity` of the blob vec?